stetho/src/main/java/com/facebook/stetho/inspector/helper/DocumentAccessRegisteredListener.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.helper;
+
+import com.facebook.stetho.inspector.elements.Document;
+
+/*
```

Comment:
The functionality is the exact same in DOM and CSS, so I think it makes sense to put it in one place.